crates/runtime/src/actor/component/logging.rs

```diff
@@ -18,7 +18,7 @@ pub mod logging_bindings {
         world: "logging",
         async: true,
         with: {
-           "wasi:logging/logging@0.2.0-rc-2023-10-18": crate::capability::logging,
+           "wasi:logging/logging@0.2.0-rc-2023-11-10": crate::capability::logging,
         },
```

Comment:
Should this just be wasi:logging/logging or is this different than the wit

Reply:
It just should not *be*, it's removed from Wasmtime and is completely ignored actually. I'll remove it